src/browser-attributes-resource.ts

```diff
@@ -1,5 +1,15 @@
 import { Resource } from '@opentelemetry/resources';
 
+type ScreenSize = 'small' | 'medium' | 'large';
+
+export const computeScreenSize = (screenWidth: number): ScreenSize => {
+  if (screenWidth <= 768) return 'small';
+  else if (screenWidth > 768 && screenWidth <= 1024) return 'medium';
+  else if (screenWidth > 1024) return 'large';
+
+  return 'large';
```

Comment:
🤔 We _should_ never get here, but maybe `unknown`  would allow us to detect/distinguish error/unexpected cases?